packages/app/src/sandbox/eval/presets/vue-cli/index.js

```diff
@@ -39,7 +40,7 @@ const getFileNameFromVm = vm => {
 export default function initialize() {
   const vuePreset = new Preset(
     'vue-cli',
-    ['vue', 'json', 'js', 'ts'],
+    ['vue', 'json', 'js', 'ts', 'coffee'],
```

Comment:
Is adding `coffee` necessary here? Seems to work with or without it.

Reply:
It's used to allow `import './test'` to resolve to `test.coffee`. I think it would be useful 😄

Reply:
Ok, cool.